libsolidity/formal/BMC.cpp

```diff
@@ -358,14 +358,6 @@ void BMC::endVisit(UnaryOperation const& _op)
 
 	switch (_op.getOperator())
```

Comment:
Since now there's only one case left we can just make it an if

Reply:
Changed.